src/JavascriptVM/vm.ts

```diff
@@ -278,6 +288,7 @@ export class BlocklyInterpreter {
   private _step(): boolean {
     let finished = false;
     this.blockHighlighted = false;
+    this.callbacks.onUnPause && this.callbacks.onUnPause();
```

Comment:
I have a question why do you think this line is needed? Don't you have to call the function `unpause` in order for `_step` to work again or I am missing something?

Reply:
If we call `unpause()` it will change the state from `PAUSED` to `RUNNING`, which is a change of current behavior and I believe we don't want that state change.